build.sbt

```diff
@@ -861,6 +869,97 @@ lazy val zioServer: ProjectMatrix = (projectMatrix in file("server/zio-http4s-se
   .jvmPlatform(scalaVersions = allScalaVersions)
   .dependsOn(zio, http4sServer, serverTests % Test)
 
+// serverless
+
+lazy val awsLambda: ProjectMatrix = (projectMatrix in file("serverless/aws/lambda"))
+  .settings(commonJvmSettings)
+  .settings(
+    name := "tapir-aws-lambda",
+    libraryDependencies ++= loggerDependencies,
+    libraryDependencies ++= Seq(
+      "com.softwaremill.sttp.client3" %% "http4s-ce2-backend" % Versions.sttp,
+      "org.http4s" %% "http4s-blaze-client" % Versions.http4s
+    )
+  )
+  .jvmPlatform(scalaVersions = allScalaVersions)
+  .dependsOn(core, cats, circeJson, awsSam)
+
+lazy val sam = Process("sam local start-api --warm-containers EAGER").run()
```

Comment:
shouldn't this be a `def`?

Reply:
maybe we can make it a `def` here and create a local `val` where it's used? otherwise it's a global value which I think is better avoided

Reply:
I use it in `settings` like:
```
.settings(
    Test / testOptions += Tests.Setup(() => {
        ...
        sam.isAlive() // starts
        ...
    }),
    Test / testOptions += Tests.Cleanup(() => sam.destroy() // kill),
  )
```
where would I have to store that local val to make it accessible in both of this places? also I take advantage of lazy to not start that process before calling sbt test

Reply:
ah indeed it is shared ... maybe sth like this?

```
Test / testOptions ++= {
  lazy val sam = ...
  Tests.Setup(() => {
        ...
        sam.isAlive() // starts
        ...
  }),
  Tests.Cleanup(() => sam.destroy() // kill),
}
```

Reply:
That worked ;)